services/movie.py

```diff
@@ -0,0 +1,43 @@
+from db.models import Movie
+from typing import List, Optional
+
+
+def get_movies(
+        genres_ids: Optional[List[int]] = None,
+        actors_ids: Optional[List[int]] = None
+) -> List[Movie]:
+    if genres_ids is None and actors_ids is None:
+        return Movie.objects.all()
+    elif genres_ids is not None and actors_ids is not None:
+        return Movie.objects.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        ).distinct()
+    elif genres_ids is not None and actors_ids is None:

```

Comment:
don't repeat your self, and just use two conditionals - one for genres, and one for actors